### PR TITLE
[DHIS2-10267] Make bold concatenation string normal style

### PR DESCRIPTION
**Pull request by jasminenguyennn**

### CODE REVIEW

File: src/components/AppStart/JSSProviderShell.component.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ insertionPoint.setAttribute('id', 'jss-insertion-point');
 document.head.appendChild(insertionPoint);
 const generateClassName = createGenerateClassName();
 const jss = create(jssPreset());
+// $FlowFixMe
```
Comment (simonadomnisoru) [inline src/components/AppStart/JSSProviderShell.component.js:16]: Hey, 
Why did you add `// $FlowFixMe` when there are no changes done to `jss`? 
Thanks!

**Review by simonadomnisoru — APPROVED** [review]

**Review by JoakimSM — APPROVED** [review]
LGTM 👍
